haskell/cabal.bzl

```diff
@@ -961,6 +974,10 @@ _stack_snapshot = repository_rule(
         "tools": attr.label_list(),
         "stack": attr.label(),
         "stack_update": attr.label(),
+        "verbose": attr.bool(
+            default = False,
+            doc = "Whether to show the output of the build",
+        ),
```

Comment:
This needs to be forwarded to the generated calls to `haskell_cabal_library` to have any effect.

Reply:
Indeed, just noticed that. Should be good now